packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -414,5 +414,39 @@ void main() {
 
     expect(value, isFalse);
   });
+  testWidgets('Switch is translucent when disabled', (WidgetTester tester) async {
```

Comment:
Small detail, leave a line between each test (to be consistent with the rest of the file).